giza_datasets/__init__.py

```diff
@@ -1,4 +1,5 @@
 from giza_datasets.hub import DatasetsHub
 from giza_datasets.loaders import DatasetsLoader
+from giza_datasets.cache_manager import CacheManager
```

Comment:
I don't think that this should be added here, as this is not something that the user would use directly

Reply:
Done. Now, the clean_cache() method is in the DatasetLoader class